src/screens/Topics/index.js

```diff
@@ -302,4 +304,4 @@ const styles = StyleSheet.create({
     textDecorationLine: 'underline'
   }
 });
-export default React.memo(Topics);
+export default Topics;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `React.memo` might have performance implications. `React.memo` is a higher order component that memoizes the result of a function component rendering, preventing unnecessary renders when the props are the same. If `Topics` component is often re-rendered with the same props, it might be beneficial to keep `React.memo`.
```diff
- export default Topics;
+ export default React.memo(Topics);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->